### ISSUE TITLE
Methods getVideoTracks and getAudioTracks does not work for remote stream

### ISSUE THREAD

**Issue by vitalevidence** [sarandogou/webrtc-everywhere#14]
I have found fix that works for me, but I'm not sure that it does not break something else:
--- ie/MediaStreamEvent.cc  
+++ ie/MediaStreamEvent.cc  
@@ -49,6 +49,7 @@
   HRESULT _hr = Utils::CreateInstanceWithRef(&_stream);
   CHECK_HR_RETURN(_hr);
   _stream->SetStream(m_Event->stream);
+_stream->SetDispatcher(const_cast<_AsyncEventDispatcher*>(GetDispatcher()));
 _hr = CComVariant(_stream).Detach(pVal);
 SafeRelease(&_stream);
 return _hr;

**Comment by RonMen** [comment]
Yes, I can confirm this. I tried webrtc-everwhere localy and run into the same issue. Is the fix working and not breaking something else for now?

Regards RonMen

**Comment by jammerhund** [comment]
Hi, same problem here. Are there any plans to fix this?

Thanks
